psychopy/data/staircase.py

```diff
@@ -858,9 +858,6 @@ def __init__(self,
             maxVal=maxVal, name=name, autoLog=autoLog)
 
         self.stopInterval = stopInterval
-
-        startVal = startVal
-        startValSd = startValSd
```

Comment:
These lines should _not_ be removed; instead, the values should be assigned to the attributes `self.startVal` and `self.startValSd`, as it probably was meant to be all along.

We should also add a unit test for this.

Reply:
I have corrected that and added two assert statements to TestQuestHandler, is that ok?